databricks-sdk-java/src/main/java/com/databricks/sdk/core/retry/RequestBasedRetryStrategyPicker.java

```diff
@@ -21,15 +21,15 @@ public class RequestBasedRetryStrategyPicker implements RetryStrategyPicker {
           // Create a new session v2.0
           new Request("POST", "/api/2.0/sql/sessions/"),
           // Delete an existing session v1.0
-          new Request("DELETE", "/api/2.0/sql/statements/sessions/*"),
+          new Request("DELETE", "/api/2.0/sql/statements/sessions/."),
```

Comment:
Does it not need to be like this for all of these?
```suggestion
          new Request("DELETE", "/api/2.0/sql/statements/sessions/.*"),
```